lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -122,6 +122,14 @@ export class PublicClientApplication {
         }
     }
 
+    /**
+     * Event handler function which allows users to fire events after the PublicClientApplication object 
+     * has loaded during redirect flows.
+     */
+    async onRedirectAppLoad(): Promise<TokenResponse> {
```

Comment:
We should document in the type that this can be null: `Promise<TokenResponse|null>`.

Reply:
And the typedoc comment shuold be updated to document why it would return null.

Reply:
And that apps should invoke it on all page loads if they are using the redirect flows and want to know when the redirect has been processed.

Reply:
done.